### ISSUE TITLE
flatpickr not defined

### ISSUE THREAD

**Issue by KaiMartin** [IceWreck/Argon-Dokuwiki-Template#36]
With the Argon template enabled I see the following error message in the javacript console of firefox:

> ReferenceError: flatpickr is not defined.

This message appears for every page of my wiki. I do not see this error message with any of the other templates I have installed (dokuwiki, material, writr).
I have yet to spot a specific failure due to this error. flatpickr seems to be related to time and date. The date in the list of old revisions seems to be fine, though.

All the best, 
---<)kaimartin(>---

**Comment by lpaulsen93** [comment]
I am also seeing this error and I assume that the Java-Script error causes the side effect that some plugins might not work, e.g. https://github.com/dokufreaks/plugin-folded/issues/68.

**Comment by SoarinFerret** [comment]
@lpaulsen93 I doubt that this is effecting the other plugins like folded, because this library is very specific to datetime pickers. I wrote a patch to include flatpickr though, you are more than welcome to test that branch out and see if it helps

**Comment by lpaulsen93** [comment]
@SoarinFerret :+1: Thanks for the quick fix. Don't get me wrong. I don't think that other plugins like folded use the datetime pickers themselves but won't the error prevent other JavaScript code from being executed/loaded?

Aynway, I will test as soon as I got time for it.

**Comment by lpaulsen93** [comment]
@SoarinFerret You are right. The flatpickr error message is gone but the folded plugin issue is still there.

**Comment by SoarinFerret** [comment]
@lpaulsen93 I may try and take a crack at getting the folded plugin issues fixed this week if I get time. I appreciate you taking an initial look for me!

**Comment by SoarinFerret** [comment]
@lpaulsen93 totally just saw you put in a pull request, nevermind!
